src/lib/models/codeObject.js

```diff
@@ -340,4 +340,8 @@ export default class CodeObject {
         return this.name;
     }
   }
+
+  get definition() {
+    return encodeURI(`${this.type}:${this.id}`);
+  }
```

Comment:
Can we name this something along the lines of `fullyQualifiedId`/`fqid`?
https://en.wikipedia.org/wiki/Fully_qualified_name

Reply:
Good Point and I fixed https://github.com/applandinc/appmap-js/pull/128/commits/c6b275659c7937dfb3dd27848b1bdf63b8065590